src/components/Description/Paragraph/index.tsx

```diff
@@ -20,11 +20,13 @@ const CITATIONS_REGEX = `(\\[(${CITATION_REGEX})+\\])`;
 const REMOVE_TRAILING_COMMA_REGEX = /,\s*(?=\])/;
 
 const xReferenceURL = {
-  cazy: 'http://www.cazy.org/fam/{}.html',
-  cog: 'https://ftp.ncbi.nih.gov/pub/COG/COG2014/static/byCOG/{}.html',
-  intenz: 'http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec={}',
-  genprop: 'https://www.ebi.ac.uk/interpro/genomeproperties/#{}',
+  cazy: 'http://www.cazy.org/{}.html',
+  cog: 'https://ftp.ncbi.nih.gov/pub/COG/COG2020/data/',
```

Comment:
```suggestion
```

As there was only one cross-reference to COG, Alex decided it was better to remove it (especially since it wasn't very informative). As we don't have COG cross-references anymore, we can just remove the key/value pair.